source/unified-test-format/tests/invalid/expectedTracingSpans-spanMalformedMissingName.yml

```
@@ -0,0 +1,16 @@
+description: "expectedTracingSpans-spanMalformedMissingName"
+
+schemaVersion: "1.26"
+
+createEntities:
+  - client:
+      id: &client0 "client0"
+
+tests:
+  - description: "missing required span name"
+    operations: []
+    expectTracingMessages:
+      client: client0
+      spans:
+        - tags:
```

```suggestion
        - tags: {}
```

If you'd like to simplify this.